packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.tsx

```diff
@@ -475,7 +475,6 @@ export class SpinButton extends React.Component<ISpinButtonProps, ISpinButtonSta
         this._updateValue(false /* shouldSpin */, this._initialStepDelay, this._onDecrement!);
         break;
       case KeyCodes.enter:
-      case KeyCodes.tab:
```

Comment:
I might be missing something but why only remove tab when the problem causing onBlur is all keyDowns?

Reply:
not sure exactly what you meant. only tabbing will cause on blur because tabbing leads to removing focus

Reply:
Ah yes that makes sense didn't think thru the visual - thanks for clarifying!